google/cloud/compute/README.md

```diff
@@ -4,6 +4,21 @@ This directory contains an idiomatic C++ client library for the
 [Compute Engine API][cloud-service-docs], a service that lets you create and run
 virtual machines on Google’s infrastructure.
 
+The Compute Engine API has a large surface and is split into multiple services,
+in different subdirectories, each interacting with a specific Compute Engine
+resource. Additionally, Compute Engine is divided into multiple libraries, one
+per resource, to reduce binary size.
+
+CMake targets follow the naming convention
```

Comment:
non-blocking for this PR: Should we also offer a `google-cloud-cpp::compute` and a `//:compute` for convenience? They would be interfaces linking to all of the `compute_*` libraries.